farcy/handlers.py

```diff
@@ -230,6 +268,16 @@ class Rubocop(ExtHandler):
     BINARY_VERSION = '0.50'
     EXTENSIONS = ['.rb']
 
+    def _prepare_directory(self, temp_dir, repo):
+        rubocop_yaml_url = "{}/contents/.rubocop.yml?ref=master".format(repo.url)  # noqa: E501
```

Comment:
Should we pass the actual PR branch name in, rather than `master`?

Reply:
We could, but do we get much value? I think this would only make a different on branches that changed the .rubocop.yml file

Reply:
Wouldn't that be exactly how you'd try to test a fix to the rubocop.yml file?

Reply:
🤔 point taken - that would be a convenient way to test fixes

I'll pass the branch down to `prepare_directory`

Reply:
Great. Consider also simply passing the entire PR object down so that if someone needs to do something else based on the PR they can. I assume repo, and branch can be determined from the PR object.